_pages/en/capabilities.md

```diff
@@ -245,6 +251,9 @@ This excerpt from to the [DORA DevOps capabilities guides](https://cloud.google.
 
 ### Section 3. CONTINUOUS TESTING
 
+*Description: Improve software quality by building reliable automates test suits and performing all kinds of testing throughout the software delivery lifecycle.
```

Comment:
I think we meant "automated" instead of "automates"? :thinking:

Reply:
fixed all the above comments, thanks